qdrant-landing/content/documentation/guides/capacity-planning.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Capacity Planning
+weight: 11
+aliases:
+  - capacity
+  - /documentation/cloud/capacity-sizing
+---
+# Capacity Planning
+
+When setting up your cluster, you'll need to figure out the right balance of **RAM** and **disk storage**. The best setup depends on a few things:
+
+- How many vectors you have and their dimensions.
+- The types of payload data you're using and their indexes.
+- What data you want to store in memory versus on disk.
+- Your cluster's replication settings.
+- Whether you're using quantization and how you’ve set it up.
+
+## Calculating RAM size 
+
+You should store frequently accessed data in RAM for faster retrieval. If you want to keep all vectors in memory for optimal performance, you can use this rough formula for estimation:
+
+```text
+memory_size = number_of_vectors * vector_dimension * 4 bytes * 1.5
+```
+
+At the end, we multiply everything by 1.5. This extra 50% accounts for metadata (such as indexes and point versions) and temporary segments created during optimization.
+
+Let's say you want to store 1 million vectors with 1024 dimensions:
+
+```text
+memory_size = 1,000,000 * 1024 * 4 bytes * 1.5 
+```
+The memory_size is approximately 6,144,000,000 bytes, or about 5.72 GB.
+
+Depending on the use case, large datasets can benefit from reduced memory requirements via quantization.
+
+## Calculating payload size
+
+This is always different. The size of the payload depends on the [structure and content of your data](/documentation/concepts/payload/?q=data+types#payload-types). For instance:
+
+- **Text fields** consume space based on length and encoding (e.g. a large chunk of text vs a few words).
+- **Integers & floats** have fixed sizes (e.g., 4 bytes for `int32` or `float32`, 8 bytes for `int64` or `float64`).
+- **Boolean fields** typically consume 1 byte. 
+
+<aside role="alert">
+    The easiest way to calculate your payload size is to use a JSON size calculator.
+</aside>
+
+Calculating total payload size is similar to vectors. We have to multiply it by 1.5 for back-end indexing processes.
+
+```text
+total_payload_size = number_of_points * payload_size * 1.5 
+```
+
+Let's say you want to store 1 million points with JSON payloads of 5KB:
+
+```text
+total_payload_size = 1,000,000 * 5KB * 1.5 
+```
+The total_payload_size is approximately 5,000,000 bytes, or about 4.77 GB.
+
+## Choosing disk over RAM
+
+For optimal performance, you should store only frequently accessed data in RAM. The rest should be offloaded to the disk. For example, extra payload fields that you don't use for filtering can be stored on disk. 
```

Comment:
I think we should word this a bit different, like:

> To achieve the best possible performance you might want everything in RAM. That is very expensive though so you're likely seeking for a balance between RAM and disk. What balance exactly depends on how you use Qdrant. Generally speaking you should put frequently accessed data in RAM, and offload other data to disk.

Quantization sounds important here as well, so I think we should describe it:

> Quantization compresses your vectors. Quantized vectors may be kept in RAM, while your original vectors are kept on disk. That may reduce your RAM consumption by up to 32x, having a significant impact on cost.

I just wrote these two sentences without thinking too much about it. If you think the above make sense, please feel free to rewrite them in something better :smile: